src/screens/Settings/FiatUnitSection.js

```diff
@@ -0,0 +1,57 @@
+/* @flow */
+import React, { PureComponent } from "react";
+import { StyleSheet, Image } from "react-native";
+import { listFiats } from "@ledgerhq/currencies";
+import LText from "../../components/LText";
+import SectionEntry from "../../components/SectionEntry";
+
+const fiatList = listFiats()
+  .reduce(
+    (acc, cur) => [
+      ...acc,
+      { value: cur.code, label: `${cur.name} (${cur.code})` }
+    ],
+    []
+  )
+  .sort((a, b) => a.label.localeCompare(b.label));
+
+class FiatUnitSection extends PureComponent<*> {
+  render() {
+    const { navigation, saveSettings, value } = this.props;
+    const arrowRight = require("../../images/arrow_right.png");
```

Comment:
dunno if it's on purpose, but should'nt we require the icon on file scope instead of render?

Reply:
no, i'm pretty sure this is the correct recommended way. https://facebook.github.io/react-native/docs/image.html
the require() in RN don't behave exactly like node.js i think. i think things gets inlined by the packager.

Reply:
I mean that it's not a problem to do it in render(), but yeah could also be done with an import, i think both are equivalent in performance

Reply:
also https://facebook.github.io/react-native/docs/images.html

Reply:
this is the same doc :D

Reply:
nope :P